### PR TITLE
init: setup for regular stories

### PR DESCRIPTION
**Pull request by tomberek**
First installment of "Tales from Nixpkgs" as well as adding a new category to the blog section.

@garbas @bryanhonof

### CODE REVIEW

File: blog/stories.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+Author: Fabian Affolter &lt;mail@fabian-affolter.ch&gt;
+Date:   Fri Mar 3 15:05:51 2023 +0100
+
+    Merge pull request #219037 from fabaff/openai-bump
+
+    python310Packages.openai: 0.26.5 -&gt; 0.27.0
+</pre>
+      <p>The original PR for this is 
+      <a href="https://github.com/NixOS/nixpkgs/pull/219037">#219037</a>. The motivation is to update the version of
+      the openai Python package. It looks like along the way there is also a related update to shell-genie and some
```
Comment (fricklerhandwerk) [inline blog/stories.xml:27]:
```suggestion
      the <code>openai</code> Python package. It looks like along the way there is also a related update to <code>shell-genie</code> and some
```

File: blog/stories.xml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+     homepage = "https://github.com/openai/openai-python";
++    changelog = "https://github.com/openai/openai-python/releases/tag/v${version}";
+     license = licenses.mit;
+     maintainers = with maintainers; [ malo ];
+   };
+</pre>
+      <p>First off, we see a bump in version from 0.26.5 to 0.27.0. Next we see the source is also updated with its
+      hash providing an integrity check. (Note: these two steps are often performed by automation, stay tuned for
+      future episodes). Lastly we see an improvement to the metadata pointing at a consistent location for the
+      changelog of the openai project.</p>
```
Comment (fricklerhandwerk) [inline blog/stories.xml:70]:
```suggestion
      changelog of the OpenAI project.</p>
```

I suppose here you mean the project, not the package as code object.

File: blog/stories.xml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      <p>Next we see changes to shell-genie. This is an application that uses openai to provide helpful command-line
```
Comment (fricklerhandwerk) [inline blog/stories.xml:71]:
```suggestion
      <p>Next we see changes to <code>shell-genie</code>. This is an application that uses <code>openai</code> to provide helpful command-line
```

File: blog/stories.xml [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      in 
+      <a href="https://github.com/dylanjcastillo/shell-genie/commit/9421a56fe353ee0066244a5cb286270a6b12cb16">this
+      addition</a>.</p>
+      <p>It is through many small changes and fixes like this that the overall ecosystem improves and these
+      improvements have benefits not just for the Nix ecosystem, but outside of it as well. A common theme we will see
+      in these tales is that our very strict approach to packaging has trickle-down benefits for upstream projects and
+      other package managers.</p>
+      <p>So we have a fairly simple update, but also improvements to metadata and collaboration with other communities.
+      Next in the PR we see some testing of the new recipe across various architectures and an approval from 
+      <a href="https://github.com/malob">malob</a>. With the approval and all checks+tests in the green, the PR was
```
Comment (fricklerhandwerk) [inline blog/stories.xml:136]:
```suggestion
      <a href="https://github.com/malob">@malob</a>. With the approval and all checks and tests in the green, the PR was
```

File: blog/stories.xml [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+      merged. Sounds like success!</p>
+      <p>Thank you 
+      <a href="https://github.com/fabaff">@fabaff</a> for your contribution.</p>
```
Comment (fricklerhandwerk) [inline blog/stories.xml:139]:
```suggestion
      <a href="https://github.com/fabaff">@fabaff</a> for your contribution!</p>
```

**Review by fricklerhandwerk — APPROVED** [review]
I love this, great stuff.

**Review by garbas — APPROVED** [review]

**Comment by garbas** [comment]
Regarding the promotion of this stories, lets wait for first 3 stories to be written and then we can start with regular promotion on Twitter/Mastodon/LinkedIn.

**Pull request merged**